website/docs/components/badge/partials/guidelines/guidelines.md

```diff
@@ -124,8 +124,19 @@ Badges come in a few icon and text combinations; text only, icon only, and icon
 <Hds::Badge @color="neutral" @type="filled" @icon="corner-down-left" @isIconOnly={{true}} @text="Icon only" />
 <Hds::Badge @color="neutral" @type="filled" @icon="hexagon" @text="Text + Icon" />
 
+## Using Badges for status
+
+Badges are commonly used to communicate status of items and objects. To avoid relying solely on color as a means to communicate status, we recommend:
+
+- Including an icon that aligns with the intended severity or importance of the status. Some common examples of this are `check` for success/positive, `alert` for warning/caution, and `x` for error/critical.
```

Comment:
[suggestion] We should consider using only the terminology that's in the component to avoid multiple terms being used to mean the same thing and help push consistent language across functions and teams.

```suggestion
- Including an icon that aligns with the intended severity or importance of the status. Some common examples of this are `check` for success, `alert` for warning, and `x` for critical.
```